docs/08-other.md

```diff
@@ -0,0 +1,6 @@
+# Other
+
```

Comment:
If someone new to the project is looking for this information, they may not be able to find under this heading. Could we link to this from elsewhere?